app/admin/schools.rb

```diff
@@ -1,28 +1,51 @@
-ActiveAdmin.register School do
+ActiveAdmin.register School do |aa|
```

Comment:
we don't use aa in the block, though, right?

Reply:
does ActiveAdmin.register School ever fail?  what happens if it does?

Reply:
@daryanypl I removed `|aa|` in https://github.com/NYPL/MyLibraryNYCApp/pull/59

I think that it safe to assume that `ActiveAdmin.register` will always succeed.  It's one of thousands of small steps in setting up the app's environment and it's convention to assume they all will pass.  I think it would take something like a typo to prevent successful initialization, and we'd expect the app's initialization to fail.  So I think we're good here.